backend/pkg/server/server.go

```diff
@@ -17,47 +17,44 @@ type Health struct {
 
 // Server represents the HTTP server
 type Server struct {
-	port int
+	port   int
+	logger logging.Logger
 }
 
 // New creates a new server instance
-func New(port int) Server {
-	return Server{port}
+func New(port int, logger logging.Logger) Server {
+	return Server{port, logger}
 }
 
 // Start starts the http server
 func (s *Server) Start() {
-	routers := createRouters()
-	logger := logging.Get("Server")
-
-	logger.Info(fmt.Sprintf("Listening on %d", s.port))
-	err := http.ListenAndServe(fmt.Sprintf(":%d", s.port), logRequest(routers))
+	routers := s.createRouters()
+	s.logger.Info(fmt.Sprintf("Listening on %d", s.port))
+	err := http.ListenAndServe(fmt.Sprintf(":%d", s.port), s.logRequest(routers))
 	if err != nil {
-		logger.Error(err)
+		s.logger.Error(err)
 	}
 }
 
-func createRouters() *mux.Router {
+func (s *Server) createRouters() *mux.Router {
 	r := mux.NewRouter()
-	r.HandleFunc("/healthcheck", healthcheckHandler)
+	r.HandleFunc("/healthcheck", s.healthcheckHandler)
 	return r
 }
 
-func logRequest(handler http.Handler) http.Handler {
+func (s *Server) logRequest(handler http.Handler) http.Handler {
 	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
-		logger := logging.Get("Server")
-		logger.Info(fmt.Sprintf("%s %s %s\n", r.RemoteAddr, r.Method, r.URL))
+		s.logger.Info(fmt.Sprintf("%s %s %s\n", r.RemoteAddr, r.Method, r.URL))
```

Comment:
Can you use the method `Infof` and avoid `Sprintf`?
https://godoc.org/github.com/sirupsen/logrus#Infof

Reply:
Done. Great suggestion!!

Reply:
This change should be done on this commit 22b1ef58fac995f16ef19a12e8c743979ed13285